src/control/lib/netdetect/netdetect.go

```diff
@@ -1075,6 +1085,14 @@ func ValidateNUMAConfig(device string, numaNode uint) error {
 	}
 	defer cleanUp(deviceScanCfg.topology)
 
+	// If the system isn't NUMA aware, skip validation
+	depth := C.hwloc_get_type_depth(deviceScanCfg.topology, C.HWLOC_OBJ_NUMANODE)
```

Comment:
dRY: looks like these 3 lines are used as a check multiple times and can be moved out to a separate function

Reply:
You are right.  I can consolidate that.